Jenkinsfile

```
@@ -197,8 +197,8 @@ spec:
             }
         }
         stage('Run the JVM Example') {
-                // Always run some sort of smoke test if this is a Pull Request
-                if (env.BRANCH_NAME.substring(0, 2) == "PR") {
+                // Always run some sort of smoke test if this is a Pull Request or from develop or master
+                if (env.BRANCH_NAME.substring(0, 2) == "PR" || env.BRANCH_NAME == "develop" || env.BRANCH_NAME == "master") {
                     // If this branch name exists in examples, use that
```

While I have recreated the master branch, my intent is to not use that branch and to use main
```suggestion
                // Always run some sort of smoke test if this is a Pull Request or from develop or main
                if (env.BRANCH_NAME.substring(0, 2) == "PR" || env.BRANCH_NAME == "develop" || env.BRANCH_NAME == "main") {
                    // If this branch name exists in examples, use that
```

---

added to next commit